README.md

```diff
@@ -65,6 +69,20 @@ and [**push-to-registry**](https://github.com/redhat-actions/push-to-registry) a
 
 For a complete example see the [example workflow](.github/workflows/example.yml).
 
+<a id="using-private-images"></a>
+
+## Using private images
+
+If your deployment requires a private image, this action will create an [image pull secret](https://docs.openshift.com/container-platform/4.2/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets) which will allow
+to reference image from secured registries.
+
+If you have already logged in to container image registry, input `create_pull_secret_from` is to specify the tool you have used to log in to container registry so that credentials file is picked up accordingly to setup pull secret, set this input to `docker` or `podman` depending upon the tool you used. This will create a pull secret using the credentials file and link the created secret to `default` service account.
```

Comment:
> weren't we going to add a feature where users can specify an already-existing pull secret?

We should do that. This is a good thing to add.

Reply:
How about input `image_pull_secret` with description
In case of private images, provide the image pull secret if you have already created that. Otherwise, see #using-private-images

Reply:
Then in the #using-private-images, we can add the required details

Reply:
`image_pull_secret_name` yes

Reply:
Okay :+1: